domains/seiko.json

```diff
@@ -5,6 +5,12 @@
         "discord": "1279377552561147964"
     },
     "record": {
-        "CNAME": "seiko-7oa.pages.dev"
+        "CNAME": "seiko-7oa.pages.dev",
+        "MX": [
+            "route1.mx.cloudflare.net.",
+            "route2.mx.cloudflare.net.",
+            "route3.mx.cloudflare.net."
+        ],
+        "TXT": "v=spf1 include:_spf.mx.cloudflare.net ~all"
```

Comment:
Can't use CNAME with any other records.

Reply:
Okay tysm

Reply:
So, Is it possible to use cloudflare page without CNAME ?